# DP: Don't define USE_XATTRS on kfreebsd and the Hurd.

--- a/Modules/posixmodule.c
+++ b/Modules/posixmodule.c
@@ -111,7 +111,7 @@
 #undef HAVE_SCHED_SETAFFINITY
 #endif
 
-#if defined(HAVE_SYS_XATTR_H) && defined(__GLIBC__)
+#if defined(HAVE_SYS_XATTR_H) && defined(__GLIBC__) && !defined(__FreeBSD_kernel__) && !defined(__GNU__)
 #define USE_XATTRS
 #endif
 
